cohorts/variant_stats.py

```diff
@@ -181,13 +207,19 @@ def variant_stats_from_variant(variant,
     """
     all_stats = []
     for (variant_file, variant_metadata) in metadata.items():
-        if ".maf" in variant_file.lower():
+        if _vcf_is_maf(variant_file=variant_file):
             stats = maf_somatic_variant_stats(variant, variant_metadata)
-        elif "strelka" in variant_file.lower():
+        elif _vcf_is_strelka(variant_file=variant_file):
             stats = strelka_somatic_variant_stats(variant, variant_metadata)
-        elif "mutect" in variant_file.lower():
+        elif _vcf_is_mutect(variant_file=variant_file):
             stats = mutect_somatic_variant_stats(variant, variant_metadata)
         else:
-            raise ValueError("Cannot parse sample fields, variant file {} is from an unsupported caller.".format(variant_file))
+            try:
```

Comment:
Here, too, my preference is checking for a particular Strelka field and a particular Mutect field, and avoiding the try/catches here, to avoid hiding other possible errors. Thoughts?